### ISSUE TITLE
Properties and classes to be deleted in DM2E Model version 1.2

### ISSUE THREAD

**Issue by kba** [DM2E/dm2e-mappings#109]
In the upcoming, final 1.2 release, we intend to delete some properties and classes since they have not been used in the data as of May 2014:

According to the latest evaluation, the unused properties are:
- dm2e:honoree
- bibo:isbn
- dm2e:misattributed
- dm2e:wasStudiedBy
- dm2e:restoration
- dm2e:owner
- edm:currentLocation
- edm:occurredAt
- edm:isSimilarTo
- dcterms:isVersionOf
- dm2e:levelOfGenesis
- bibo:issn
- dm2e:influencedBy
- rdaGr2:dateOfEstablishment
- dm2e:condition
- dcterms:medium
- geo-pos:long
- dm2e:patron
- dm2e:writtenAreaDimension
- ore:describes
- rdaGr2:gender
- dm2e:wasTaughtBy
- dm2e:studentOf
- edm:happenedAt
- edm:wasPresentAt
- ore:aggregates
- rdaGr2:dateOfTermination
- rdaGr2:professionOrOccupation
- dm2e:portrayed
- dm2e:originb
- dm2e:modeOfAcquisition

The unused classes are:
- dm2e:Work
- fabio:Cover
- dm2e:Photo
- dm2e:Publication
- vivo:Museum
- vivo:University
- edm:Event
- dm2e:File
- dm2e:Document
- fabio:Article
- foaf:Document
- dm2e:Archive
- dm2e:Collection
- vivo:Library
- foaf:Image

**Comment by d0rg0ld** [comment]
Can you state the exact date of the analysis? Was the dataset from SBB already included?

**Comment by kba** [comment]
Data was generated on April, 29th 2014, 17:04. From SBB we included DTA  and kpe (http://data.dm2e.eu/data/dataset/sbb/kpe/20140417133253835).

But I don't think all those properties/classes listed here are actually marked for deletion in the specs/OWL. @edroege @JIwanowa ?

**Comment by edroege** [comment]
Candidates to be removed from the model in the draft of the current revision are (order as in the model specification):

edm:ProvidedCHO properties
- dm2e:origin
- dm2e:modeOfAcquisition
- dm2e:levelOfGenesis
- dm2e:condition
- dm2e:restoration
- dm2e:writtenAreaDimension
- dm2e:wasStudiedBy
- dm2e:wasTaughtBy
- dm2e:honoree
- dm2e:misattributed
- dm2e:patron

edm:PhysicalThing subclasses
- dm2e:Document

edm:Agent properties
- skos:hiddenLabel

foaf:Person properties
- dm2e:influencedBy
- dm2e:studentOf

foaf:Organization subclasses
- dm2e:Archive
- vivo:Library
- vivo:Museum
- vivo:University

edm:NonInformationResource subclasses
- edm:Event

skos:Concept properties
- skos:broader
- skos:narrower

skos:Concept subclasses
- dm2e:Work
- dm2e:Publication

edm:Place properties
- wgs84_pos:lat
- wgs84_pos:long
- wgs84_pos:alt

edm:Event: Class with all properties 
- edm:happenedAt
- edm:occurredAt
- skos:prefLabel
- skos:altLabel
- skos:note

**Comment by d0rg0ld** [comment]
I would at least leave the "standard" properties such as the wgs84_pos props for edm:Place and the skos properties (broader/narrower)

**Comment by d0rg0ld** [comment]
I can also confirm that I will use dm2e:modeOfAcquisition in the next iteration of our codices mapping

**Comment by d0rg0ld** [comment]
For JDC we will most likely use dm2e:Document and dm2e:Archive

**Comment by edroege** [comment]
@d0rg0ld Good to know that you will use some of the classes and a property!

Can you explain what you mean with "standard" properties? We do not have all edm properties in our model. It would be nice if providers would use the properties for exact place coordinates but maybe thi is just not possible when we deal with old documents? Why are broader and narrower standard if no one uses them in the mappings?

**Comment by d0rg0ld** [comment]
Well I dont know if it is so good to build the model "strictly empirically" by only looking at the properties used at a certain point in time. New providers could show up with data covering properties that have not been used before. With "standard" properties I therefore indeed mean the properties that belong to "external" classes incorporated into the DM2E model. IMHO they should always be legal to use.

**Comment by edroege** [comment]
That is true. You have maybe noticed that not every unused property was marked for the deleting-discussion (e.g. dm2e:File that may be used for archival items) so it is not strictly empirical. But there are a lot of classes and properties that were not used even though they could have been used if you just think of the provided data. For example, subclasses of foaf:Organization were not used at all even though we already have mapped data from libraries and universities. My thought when marking dm2e:Archive was that mappings with archival data would also not use subclasses of foaf:Organization. Why should this be different when data from archives is mapped compared to data from libraries?

If we can reduce the complexity of the model, we have the chance to do it now based on the evaluations. If we have a reason to keep the classes and properties that were not used (like when you say that you are planing to use them in future) that is totally fine and we should keep them. But we should get rid of the ballast of the model to make it easier to use once we cannot always provide additional explanations. Makes the reuse by others much easier!

**Comment by JIwanowa** [comment]
We have left all classes and properties that are intended to be used during the upcoming DM2E mappings in the last draft of the model.

**Comment by d0rg0ld** [comment]
Sorry for being late: I just talked to Mariana Damova today, she is about to map the contribution of the Bulgarian ACademiy of sciences during the next days. I think it would be good to wait for her if possible at least regarding the CHO props:

edm:ProvidedCHO properties

dm2e:origin
dm2e:modeOfAcquisition
dm2e:levelOfGenesis
dm2e:condition
dm2e:restoration
dm2e:writtenAreaDimension
dm2e:wasStudiedBy
dm2e:wasTaughtBy
dm2e:honoree
dm2e:misattributed
dm2e:patron

What I know is that she will use dm2e:Work as skos:concept subclass !

**Comment by knepper** [comment]
I just populated edm:currentLocation in my last mapping. It could be nice for maps and might be easy to add if the dataprovider has digitized own holdings.
